src/actions/signUpActions.js

```diff
@@ -0,0 +1,11 @@
+import { REQUEST_IN_PROGRESS } from './index'
+import axios from 'axios';
+
+export const createUser = userInfo => dispatch => {
+    dispatch({
+        type: REQUEST_IN_PROGRESS
+    })
+    return axios.post('https://api-plants.herokuapp.com/auth.register', userInfo)
+    .then(res => console.log(res))
```

Comment:
registration should be handled similar to login on the frontend ... you receive a token that needs to be stored in localStorage ... and you use the req.data.user to update the redux store user data